daemon.py

```diff
@@ -154,7 +154,6 @@ def daemonize(self):
 
 
     def start(self, foreground=False):
-        log.info("Starting")
```

Comment:
Indeed, it works for both the collector and dogstatsd, I'm going to let this one and remove the duplicate in the collector (`Running in foreground`).